kubernetes-crew/terraform/main.tf

```diff
@@ -56,143 +119,128 @@ resource "kubiya_knowledge" "kubernetes_troubleshooting" {
   description      = "Common issues and resolution procedures"
   labels           = ["kubernetes", "troubleshooting"]
   supported_agents = [kubiya_agent.kubernetes_crew.name]
-  content          = file("${path.module}/knowledge/kubernetes_troubleshooting.md")
+  content          = data.http.kubernetes_troubleshooting.response_body
 }
 
-# Load prompts from files
-locals {
-  health_check_prompt     = file("${path.module}/prompts/health_check.md")
-  resource_check_prompt   = file("${path.module}/prompts/resource_check.md")
-  cleanup_prompt          = file("${path.module}/prompts/cleanup.md")
-  network_check_prompt    = file("${path.module}/prompts/network_check.md")
-  security_check_prompt   = file("${path.module}/prompts/security_check.md")
-  backup_check_prompt     = file("${path.module}/prompts/backup_check.md")
-  cost_analysis_prompt    = file("${path.module}/prompts/cost_analysis.md")
-  compliance_check_prompt = file("${path.module}/prompts/compliance_check.md")
-  update_check_prompt     = file("${path.module}/prompts/update_check.md")
-  capacity_check_prompt   = file("${path.module}/prompts/capacity_check.md")
-  upgrade_check_prompt    = file("${path.module}/prompts/upgrade_check.md")
-  scaling_check_prompt    = file("${path.module}/prompts/scaling_check.md")
-}
 
 # Core Health Check Task
 resource "kubiya_scheduled_task" "health_check" {
   count          = var.enabled_tasks.health_check ? 1 : 0
-  scheduled_time = var.task_schedules.health_check.start_time
-  repeat         = var.task_schedules.health_check.repeat
+  scheduled_time = try(var.task_schedules.health_check.start_time, "2024-11-3 08:00:00")
+  repeat         = try(var.task_schedules.health_check.repeat, "daily")
   channel_id     = var.notification_slack_channel
   agent          = kubiya_agent.kubernetes_crew.name
-  description    = local.health_check_prompt
+  description    = data.http.health_check_prompt.response_body
 }
 
 # Resource Optimization Task
 resource "kubiya_scheduled_task" "resource_check" {
   count          = var.enabled_tasks.resource_check ? 1 : 0
-  scheduled_time = var.task_schedules.resource_check.start_time
-  repeat         = var.task_schedules.resource_check.repeat
+  scheduled_time = try(var.task_schedules.resource_check.start_time, "2024-11-3 08:00:00")
```

Comment:
I wonder if we can change the date to current time since in my opinion this is too strict to November 3rd
That related to all the scheduled time in this file. 

@Kubiya-Barak